frontend/src/store/store.ts

```diff
@@ -1085,23 +1098,57 @@ export default new Vuex.Store<IState>({
       await dispatch('fetchSkillBalance');
     },
 
-    async fetchCharacters({ dispatch }, characterIds: (string | number)[]) {
-      await Promise.all(characterIds.map(id => dispatch('fetchCharacter', { characterId: id })));
+    async fetchCharacters({ state, dispatch }, characterIds: (string | number)[]) {
+      const { Characters } = state.contracts();
+      if (!Characters) return;
+
+      console.log('fetch 1');
+      console.log('address: ', Characters?.options.address);
+      console.log('getNFTCall: ', getNFTCall(charactersAbi, Characters?.options.address, 'get', characterIds.map(characterId => [characterId])));
+      const multiCharacterDatas: string[] = await dispatch(
+        'multicall',
+        getNFTCall(charactersAbi, Characters?.options.address, 'get', characterIds.map(characterId => [characterId])));
```

Comment:
I don't think we need optional chaining operator "?", as at the start we have ` if (!Characters) return;`